### ISSUE TITLE
GFX doesn't go back to Integrated after being forced into Discrete

### ISSUE THREAD

**Issue by jlhopes** [steveschow/gfxCardStatus#26]
First off, Steve, thank you for your work on this! I'm now running 2.4.4i, and it's helped a lot with the annoying flickering I get on my 2015 MBP in certain apps (mainly Word 2011). I'm running Sierra (10.12.6), and half the time I use an external monitor, which forces the computer (and gfx) into discrete graphics mode. However, when I unplug from the external monitor and go back to using the MBP screen, gfx doesn't switch back to integrated mode, and when I try to do it manually, the "d" stays in place. I have to quit and restart the program to get it working properly. Not sure if there is any way to fix this, but if there is it would be amazing. Thanks!

**Comment by bioharz** [comment]
Yes. You have quit all programs that were using the dGPU. You can'T fix taht with gfxCardStatus. That's a MacOS issue.
